rhizome/lib/vm_setup.rb

```diff
@@ -487,9 +524,13 @@ def forwarding
     r("ip netns exec #{q_vm} sysctl -w net.ipv4.ip_forward=1")
   end
 
-  def install_systemd_unit(max_vcpus, cpu_topology, mem_gib, vhost_sockets)
+  def install_systemd_unit(max_vcpus, cpu_topology, mem_gib, vhost_sockets, nics)
     cpu_setting = "boot=#{max_vcpus},topology=#{cpu_topology}"
 
+    tapnames = nics.map { |net6, net4, tap, mac|
+      "-i #{tap} "
```

Comment:
Is this a subtle way to get a trailing space of a bit of extra safety when being spliced into a command line? Otherwise you could use `join(' ')`

Reply:
changed it as suggested